package/cfg/rke2-cis-1.23-hardened/policies.yaml

```
@@ -93,7 +93,7 @@ groups:
 
       - id: 5.2.2
         text: "Minimize the admission of privileged containers (Manual)"
-        audit: "kubectl get psp global-restricted-psp -o json | jq -r '.spec.runAsUser.rule'"
+        audit: kubectl get psp global-restricted-psp && kubectl get psp global-restricted-psp -o json | jq -r ".spec.runAsUser.rule" || kubectl get psp restricted-psp && kubectl get psp restricted-psp -o json | jq -r ".spec.runAsUser.rule" || kubectl get psp unrestricted-psp && kubectl get psp unrestricted-psp -o json | jq -r ".spec.runAsUser.rule" || kubectl get psp restricted-noroot-psp -o json | jq -r ".spec.runAsUser.rule"
```

I wonder if there is any reason for not checking whether the `restricted-noroot-psp` exists or not? 

```suggestion
        audit: kubectl get psp global-restricted-psp && kubectl get psp global-restricted-psp -o json | jq -r ".spec.runAsUser.rule" || kubectl get psp restricted-psp && kubectl get psp restricted-psp -o json | jq -r ".spec.runAsUser.rule" || kubectl get psp unrestricted-psp && kubectl get psp unrestricted-psp -o json | jq -r ".spec.runAsUser.rule" || kubectl get psp restricted-noroot-psp && kubectl get psp restricted-noroot-psp -o json | jq -r ".spec.runAsUser.rule"
```

---

Yeah makes sense. Updated the PR